### DISCUSSION TITLE
Update of getdown documentation (README file)

### DISCUSSION THREAD

**Discussion by serge-xav** [bekoenig/getdown#6] [General]
Hi, I see in the README file, that there is still a link to the WIKI of the threerings/getdown GitHub. That can be easily updated to the wiki of the current project. 
There is also still a link to the [OOO Libs Google group](http://groups.google.com/group/ooo-libs) which is now filled with Spam. I propose to completely remove this link if you agree. I will create pull request for this modification of Documentation if it's okay for you.
Xavier

**Comment by bekoenig** [comment]
Thank you for your fix.

I think the wiki need some clean up and update too, but I will focus on an improved test coverage.

**Comment by serge-xav** [comment]
Hi again, do you know if anyone has still administrator rights on the [OOO Libs Google Group](http://groups.google.com/group/ooo-libs) ?
@samskivert maybe ? The group should at least be put in Read-Only mode, and the 3 first pages of Spam to be removed.

**Comment by samskivert** [comment]
I marked all those posts as spam and turned on moderation for new user posts. I see all messages from that mailing list, so it's very weird that none of those spam messages were sent to me via email. Maybe they were and then were marked as spam by gmail, so I never saw them. Anyhow, hopefully the new system of moderating a new user's first post will help keep it from happening again.
